cypress/integration/search.spec.js

```diff
@@ -38,13 +38,13 @@ context('Searching', () => {
   })
 
   describe('search results should be ranked by modified levenshtein distance', () => {
-    it('should show nipa before nipaw if the search string is nipa', () => {
-      // on the previous implementation nipaw is showed first
-      cy.visit('/search/nipa')
+    it('should show nipîhk before nîpîhk if the search string is the former', () => {
+
+      cy.visit('/search/nipîhk')
 
       cy.get('[data-cy=search-results]').first()
-        .should('contain', 'nipa')
-        .and('contain', 'Means, during the night')
```

Comment:
nipa here is a MD only preverb. It's not shown anymore

Reply:
@Madoshakalaka The preverbs were not part of Katie's original comparison.

If preverbs are merged, we always use the spelling from CW, and as far as I can judge, the meanings are considered similar but not equivalent, i.e. having the two meanings of two separate lines, followed by MD and CW, respectively.

Of the potentially mergeable ones:

1. CW has _nîpâ-_ (IPV) 'at night, in the dark', so that is mergeable with the MD preverb case.

2. The second person prefix _ki-_ in MD does not match the past tense preverb _kî-_, so that _cannot_ be merged, and the MD version should be presented as separate dictionary entries (spelled _ki-_ in MD and _kî-_ in CW).

3. The following do **NOT** have matching meanings, and would have to be kept as separate dictionary entries with the own headwords and source indications:
_kici-_

4. Other observations - the following are not preverbs, but should not be merged:

a. _nici_ (MD, misspelled, should be: _nîci-_) should not be merged with CW, as the MD one is actually the prefix _nîci-_, and the latter is an independente particle _nici_.

b. wîci- has different meanings in MD vs. CW.

c. wa-/wâ- have different meanings in MD vs. CW.

d. _ta-_ has different meanings in MD vs. CW.

5. Of the ones listed for manual verification:

a. _maci-_ (bad) is different from _mâci-_ (begin), so they should **not** be merged.

b.  _not_ to be merged: ['âkô-', 'ako-'],  ['âpihci-', 'apihci-'], ['âsê-', 'asê-'], ['ka-', 'kâ-'], ['kîpi-', 'kipi-'], ['kwataki-', 'kwâtaki-'], ['pîmi-', 'pimi-'] and['tâhci-', 'tahci-'].

c. can be merged: ['kisiwi-', 'kisîwi-']